### DISCUSSION TITLE
Add support for Crypto.com | DeFi Wallet -> WalletConnect doesn't work anymore!

### DISCUSSION THREAD

**Discussion by destructioneth** [rainbow-me/rainbowkit#1602] [Ideas]
Speaking on behalf of all the Cronos devs, we love RainbowKit and would love to have this! 

I searched similar discussions and they mention WalletConnect works, but this is not the case anymore with the newer versions.

In fact, a lot of Cronos projects have built our frontend with RainbowKit, because WalletConnect used to work smoothly but now it doesn't and now our users can't really connect using Defi Wallet anymore.

Here are all the docs:

https://crypto-com.github.io/deficonnect-monorepo/

Please make this happen 🤖

**Comment by Oyebob** [comment]


**Reply by destructioneth** [reply]
Scam

**Comment by 0xmkh** [comment]
@destructioneth We should be able to support crypto.com + all other wallets that support EIP6963 in the RainbowKit V2 release
